client.go

```diff
@@ -63,12 +107,32 @@ func (req *Request) Execute(method, url string) (*Response, error) {
 	if err != nil {
 		return nil, err
 	}
+	for k, _ := range req.QueryParam {
+		req.QueryParam.Del(k)
+	}
```

Comment:
This seems not needed - do the tests fail if removed?

Reply:
This is the fix to the problem we had earlier, where the query parameters were appended to the URL twice